mealy/error_visualizer.py

```diff
@@ -151,12 +153,18 @@ def plot_error_tree(self, size=None):
                         lte_split_without_feature = lte_split[0].split('\\n')[0]
                         lte_split_with_new_feature = lte_split_without_feature + '\\n' + split_feature
                         lte_modified = ' != '.join([lte_split_with_new_feature, str(descaled_value)])
-                    new_label = '\\nentropy'.join([lte_modified, entropy_split[1]])
+                    new_label = lte_modified
+                else:
+                    samples_split = node_label.split('\\nsamples')
+                    new_label = samples_split[0]
 
-                global_error = float(wrongly_predicted_samples[idx]) / n_total_errors
-                local_error = float(wrongly_predicted_samples[idx]) / (
-                            well_predicted_samples[idx] + wrongly_predicted_samples[idx])
-                new_label += '\\nglobal error = %.3f %%\\n' % (global_error * 100)
+                value_split = samples_split[1].split('\\nvalue')
+                class_split = value_split[1].split('\\nclass')
+
+                new_label += '\\nsamples' + value_split[0] + \
+                             '\\nclass' + class_split[1] + \
+                             '\\nlocal error = %.3f %%' % (local_error * 100) + \
+                             '\\nglobal error = %.3f %%\\n' % (global_error * 100)
```

Comment:
We might want to use `Fraction of total error` instead of global error? (hoping local error would be clear enough)

Reply:
I'll do a separate PR of rewording global error into fraction of total error.

Reply:
Seems wiser :)